flink-formats/flink-avro-confluent-registry/src/test/java/org/apache/flink/formats/avro/registry/confluent/RegistryAvroFormatFactoryTest.java

```diff
@@ -127,6 +139,66 @@ public void testMissingSubjectForSink() {
         createTableSink(SCHEMA, options);
     }
 
+    @Test
+    public void testDeserializationSchemaWithOptionalProperties() {
+        final AvroRowDataDeserializationSchema expectedDeser =
```

Comment:
Reduce code duplication and move shared parts with the test above into a helper method.